main.cpp

```diff
@@ -84,6 +84,7 @@ int showHelp() {
            "    dockd --config [docked|undocked]    - write config files\n"
            "    dockd --set [docked|undocked]       - set the saved config\n"
            "    dockd --daemon                      - start the dock daemon\n");
+    return EXIT_FAILURE;
```

Comment:
Thanks for your contribution. Would you change this to `EXIT_SUCCESS`? Because this function is called with `--help` so it should not exit with 1. It's legitimate to request `--help` ;)

Reply:
Sounds reasonable :) Fixed!